examples/plot_tikhonov.py

```diff
@@ -0,0 +1,247 @@
+            this_fixation_spatial_features[probe_bin, :] = basis_projection
+
+    # Count spikes in window of interest relative to fixation
+    bins = fixation_time + bin_template
+    searchsorted_idx = np.searchsorted(spiketimes,
+                                       [fixation_time + window[0],
+                                        fixation_time + window[1]])
+    this_fixation_spike_counts = np.histogram(spiketimes[searchsorted_idx[0]:
+                                                         searchsorted_idx[1]], bins)[0]
+
+    # Accumulate
+    fixation_id = np.concatenate((fixation_id, this_fixation_id), axis=0)
+    spatial_features = np.concatenate((spatial_features, this_fixation_spatial_features), axis=0)
+    spike_counts = np.concatenate((spike_counts, this_fixation_spike_counts), axis=0)
+
+    # Zero pad
+    spatial_features = np.concatenate((spatial_features, np.zeros((n_zero_bins, n_spatial_basis))), axis=0)
+    fixation_id = np.concatenate((fixation_id, -999. * np.ones(n_zero_bins)), axis=0)
+
+# Convolve with temporal basis
+features = strf_model.convolve_with_temporal_basis(spatial_features, temporal_basis)
+
+# Remove zeropad
+features = features[fixation_id != -999.]
+
+########################################################
+# Visualize the distribution of spike counts
+
+########################################################
+
+plt.hist(spike_counts, 10)
+plt.show()
+
+########################################################
+# Plot a few rows of the design matrix
+
+########################################################
+
+plt.imshow(features[30:150,:], interpolation='none')
+plt.show()
+
+#################################################################
+# Design prior covariance matrix for Tikhonov regularization
+
+#################################################################
+
+prior_cov = strf_model.design_prior_covariance(sigma_temporal=3., sigma_spatial=5.)
+
+plt.imshow(prior_cov, cmap='Greys', interpolation='none')
+plt.colorbar()
+plt.show()
+
+np.shape(prior_cov)
+
+########################################################
+# Fit models
+
+########################################################
+
+from sklearn.cross_validation import train_test_split
+Xtrain, Xtest, Ytrain, Ytest = train_test_split(features, spike_counts, test_size=0.2, random_state=42)
+
+########################################################
+
+from pyglmnet import utils
+n_samples = Xtrain.shape[0]
+Tau = utils.tikhonov_from_prior(prior_cov, n_samples)
+
+glm = GLM(distr='poisson', alpha=0., Tau=Tau, score_metric='pseudo_R2')
+glm.fit(Xtrain, Ytrain)
+cvopt_lambda = glm.score(Xtest, Ytest).argmax()
+print("train score: %f" % glm[cvopt_lambda].score(Xtrain, Ytrain))
+print("test score: %f" % glm[cvopt_lambda].score(Xtest, Ytest))
+weights = glm[cvopt_lambda].fit_['beta']
+
+########################################################
+# Visualize
+
+########################################################
```

Comment:
same here. `sphinx_gallery` automatically separates the code blocks from the comment block ...